README.md

```diff
@@ -22,16 +22,31 @@ To view the contents of the database, install pgAdmin. Once your account is set
 ## Running database migrations
 **If you are creating a new table, the table name should be singular, i.e. "student" not "students" for consistency.**
 1. Install PostgreSQL and create a database.
+Run ```psql postgres``` to go to the psql shell.
+Run ```CREATE acmwosu``` to create a new database called "acmwosu".
```

Comment:
This can also be done in pgAdmin but this probably isn't wrong so just add "Run ... or go to pgAdmin to create..."